boards/neokey_trinkey/Cargo.toml

```diff
@@ -0,0 +1,74 @@
+[package]
+name = "neokey_trinkey"
+version = "0.1.0"
+authors = ["Broderick Carlin <broderick.carlin@gmail.com>"]
+description = "Board Support crate for the Adafruit Neokey Trinkey"
+keywords = ["no-std", "arm", "cortex-m", "embedded-hal"]
+license = "MIT OR Apache-2.0"
+repository = "https://github.com/atsamd-rs/atsamd"
+readme = "README.md"
+edition = "2018"
+
+[dependencies]
+cortex-m = "0.6"
+embedded-hal = "0.2.3"
+nb = "0.1"
+
+[dependencies.cortex-m-rt]
+version = "0.6.12"
+optional = true
+
+[dependencies.atsamd-hal]
+path = "../../hal"
+version = "0.13"
+default-features = false
+
+[dependencies.usb-device]
+version = "0.2"
+optional = true
+
+[dependencies.usbd-serial]
+version = "0.1"
+optional = true
+
+[dependencies.panic-halt]
+version = "0.2"
+optional = true
+
+[dependencies.ws2812-timer-delay]
+version = "0.3.0"
+features = ["slow"]
+optional = true
+
+[dependencies.smart-leds]
+version = "0.3.0"
+optional = true
+
+[dev-dependencies]
+panic-halt = "0.2"
+
+[features]
+# ask the HAL to enable atsamd21e support
+default = ["rt", "atsamd-hal/samd21e"]
+leds = ["ws2812-timer-delay", "smart-leds"]
+panic_halt = ["panic-halt"]
+rt = ["cortex-m-rt", "atsamd-hal/samd21e-rt"]
+unproven = ["atsamd-hal/unproven"]
+use_semihosting = []
```

Comment:
Could we update the examples to key off of this feature to do different panic behavior like [other examples](https://github.com/atsamd-rs/atsamd/blob/1f7517177beb8ccd08748a19b12bf1cbf683fe23/boards/feather_m4/examples/neopixel_rainbow.rs#L16)

I'm also open to just removing this feature